lib/modules/manager/fleet/extract.ts

```diff
@@ -50,6 +52,24 @@ function extractFleetHelmBlock(doc: FleetHelmBlock): PackageDependency {
       skipReason: 'missing-depname',
     };
   }
+
+  if (isOCIRegistry(doc.chart)) {
+    const ociImageName = doc.chart.replace('oci://', '').split('/').at(-1);
+    const ociRegistryPath = doc.chart
+      .replace('oci://', '')
+      .split('/')
+      .slice(0, -1)
+      .join('/');
+
+    dep.datasource = DockerDatasource.id;
+    dep.depName = ociImageName;
+    dep.currentValue = doc.version;
+    dep.packageName = `${ociRegistryPath}/${ociImageName}`;
```

Comment:
simplify remove the `oci://` prefix 

https://github.com/renovatebot/renovate/blob/2d9305f250106c15db4bd390b4f239cc00c58776/lib/modules/manager/kustomize/extract.ts#L150

see there for a good implementation